### PR TITLE
Docs reformatting with :guilabel:

### PR DESCRIPTION
**Pull request by rexzing**
Refactoring the docs by replacing Admin menu guidance with :guilabel: sphinx role for all the docs. This is a fix to issue #5137

### CODE REVIEW

**Comment by dojutsu-user** [comment]
Thanks @rexzing :tada: 
Someone from the core team will review this.

File: docs/advertising/ethical-advertising.rst [hunk 1]
```diff
@@ -154,15 +154,14 @@ or a `Supporter <https://readthedocs.org/sustainability/#donations>`_.
...
+* Go to the drop down user menu in the top right of the Read the Docs dashboard and clicking :guilabel:`Settings` (https://readthedocs.org/accounts/edit/).
+* On the :guilabel:`Advertising` tab, you can deselect **See paid advertising**.
 
 Project owners can also opt out of paid advertisements for their projects.
 You can change these options:
 
-* Click on your **Project** page (`/projects/<slug>/`)
-* Click the  **Admin** dashboard link 
-* Choose the **Advertising** submenu on the left side
+* Click on your :guilabel:`Project` page (`/projects/<slug>/`)
```
Comment (stsewd) [inline docs/advertising/ethical-advertising.rst]: We don't have a `Project` button here.

Comment (stsewd) [reply docs/advertising/ethical-advertising.rst]: We can rewrite this as `go to your  project page..` I think

File: docs/guides/build-notifications.rst [hunk 2]
```diff
@@ -9,8 +9,8 @@ This makes sure you know when your builds have failed.
 
 Take these steps to enable build notifications using email:
 
-* Go to **Admin > Notifications** in your project.
-* Fill in the **Email** field under the **New Email Notifications** heading
+* Go to :guilabel:`Admin` > :guilabel:`Notifications` in your project.
+* Fill in the **Email** field under the :guilabel:`New Email Notifications` heading
```
Comment (stsewd) [inline docs/guides/build-notifications.rst]: New email notifications isn't a button neither.

File: docs/guides/build-notifications.rst [hunk 3]
```diff
@@ -22,8 +22,8 @@ Read the Docs can also send webhooks when builds fail.
 
 Take these steps to enable build notifications using a webhook:
 
-* Go to **Admin > Notifications** in your project.
-* Fill in the **URL** field under the **New Webhook Notifications** heading
+* Go to :guilabel:`Admin` > :guilabel:`Notifications` in your project.
+* Fill in the **URL** field under the :guilabel:`New Webhook Notifications` heading
```
Comment (stsewd) [inline docs/guides/build-notifications.rst]: Same `New webhook notifications`

File: docs/webhooks.rst [hunk 4]
```diff
@@ -90,7 +89,7 @@ Using the generic API integration
 For repositories that are not hosted with a supported provider, we also offer a
 generic API endpoint for triggering project builds. Similar to webhook
 integrations, this integration has a specific URL, found on the project's
-**Integrations** Admin dashboard page on readthedocs.org.
+:guilabel:`Admin` > :guilabel:`Integrations` dashboard page on readthedocs.org.
```
Comment (stsewd) [inline docs/webhooks.rst]: I think we need to modify the above sentence `...found on the project's`

File: docs/webhooks.rst [hunk 5]
```diff
@@ -109,7 +108,7 @@ branches
 
 token
     The integration token. You'll find this value on the project's
-    **Integrations** Admin dashboard page.
+    :guilabel:`Admin` > :guilabel:`Integrations` dashboard page.
```
Comment (stsewd) [inline docs/webhooks.rst]: Same here, above sentence should be changed.

**Comment by rexzing** [comment]
Sure... I will do these changes.

**Comment by rexzing** [comment]
Should I close this PR and send a new one or is there a way to add changes to this PR?

**Comment by dojutsu-user** [comment]
You can checkout on this branch, make and commit changes and then push them.
It will appear here.
No need of new PR.

File: docs/webhooks.rst (hunk 4 above)
Comment (rexzing) [reply docs/webhooks.rst]: :guilabel:`Admin` > :guilabel:`Integrations` found on the project's dashboard page.

File: docs/webhooks.rst (hunk 5 above)
Comment (rexzing) [reply docs/webhooks.rst]: The integration token found on the project's **Integrations**  dashboard page. (:guilabel:`Admin` > :guilabel:`Integrations`)

File: docs/webhooks.rst (hunk 4 above)
Comment (rexzing) [reply docs/webhooks.rst]: Similar to webhook integrations,
this integration has a specific URL, which can be found on the project's Integrations dashboard page.
(:guilabel:`Admin` > :guilabel:`Integrations`).

**Comment by rexzing** [comment]
I added the requested changes

File: docs/webhooks.rst [hunk 6]
```diff
@@ -87,9 +87,9 @@ Using the generic API integration
 ---------------------------------
 
 For repositories that are not hosted with a supported provider, we also offer a
-generic API endpoint for triggering project builds. Similar to webhook
-integrations, this integration has a specific URL, found on the project's
-:guilabel:`Admin` > :guilabel:`Integrations` dashboard page on readthedocs.org.
+generic API endpoint for triggering project builds. Similar to webhook integrations,
+this integration has a specific URL, which can be found on the project's **Integrations** dashboard page.
```
[the remark this answers is not in the document]
Comment (rexzing) [reply docs/webhooks.rst]: Sorry I didn't get it. Please explain little more.

[the remark this answers is not in the document]
Comment (stsewd) [reply docs/webhooks.rst]: `dashboard page. (:guilabel:Admin > :guilabel:Integrations).`

We don't need a period before `(`

**Pull request merged**